.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
Could you please remove .idea folder from pull request.
I think this is your editor-specific config.